### PR TITLE
Feature/issue209 original list

### LINKED ISSUE #209: Not possible to return to original unclassified list after selecting a classification in PxWeb
**Describe the bug**
When selecting a classification from the drop down menu in PxWeb (Choose variables section), it is not possible to return to the original unclassified list (turns out empty). 

**To Reproduce**
1. Go to https://stat.hel.fi/pxweb/en/Aluesarjat/Aluesarjat__kou__pksoa/A01S_HKI_Vaesto_koulutusaste.px/
2. Click on 'Select classification' and choose any alternative (except Select classification)
3. Click on Select classification' to try to return to the original list
4. See error, which is an empty list

**Expected behavior**
Step number 4 should return the original unclassified list, not an empty list.

**Actual behavior**
Step number 4 should returns an empty list list.

**Screenshots**
image
image
image

**Environment information**
 - PxWeb version: 2021 v1 1.0
 - OS: Windows 10
 - Browser: chrome
 - Device: PC

### PR DESCRIPTION
**Pull request by MikaelNordberg**
Restore variable values by selecting "--Select classification--" in the valueset/aggregation dropdown.

NOTE: This PR is dependent on the "Restore variable values with valueset _ALL_ in PxFileBuilder" (https://github.com/statisticssweden/PCAxis.Core/pull/14) pull request in the PCAxis.Core repository.

Closes #209

### CODE REVIEW

**Comment by likp** [comment]
@MikaelNordberg I will wait with my review until you have upgraded the Nuget packages for PCAxis.Core.

File: PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelectCodebehind.vb [hunk 1]
```diff
@@ -406,9 +407,9 @@ Public Class VariableSelectorValueSelectCodebehind
 
         'Match grouping selected in the interface. Use of backbutton may have caused missmatches
         'between model on server and sclections sent by the interface.
-        If GroupingDropDown.SelectedValue.Length > 0 Then
-            Marker.SelectedGrouping = GroupingDropDown.SelectedValue
-        End If
+        'If GroupingDropDown.SelectedValue.Length > 0 Then
```
Comment (likp) [inline PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelectCodebehind.vb:410]: Remove the code instead of comment it out.

Comment (MikaelNordberg) [reply PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelectCodebehind.vb]: Comments removed

**Review by likp — APPROVED** [review]
Looks good!

**Pull request merged**